src/hatti/charting.cljc

```diff
@@ -73,7 +80,8 @@
    metadata of this above value would be:
    {:bins ['1 to 2', '3 to 4', '5 to 6', '7 to 8', '9 to 10']}"
      [answers bins typ]
-     (let [numbers (map (str->int typ) answers)
+     (let [numbers (->> answers
```

Comment:
You're right. It isn't. It's left over from an earlier commit. I deleted the last form and forgot to modify the rest of it appropriately.